rpc/serializers.py

```diff
@@ -847,6 +854,63 @@ def get_rfc_number(self, clustermember: ClusterMember) -> int | None:
 
         return rfctobe["rfc_number"] if rfctobe else None
 
+    def get_disposition(self, clustermember: ClusterMember) -> str | None:
+        """Get the disposition slug for this cluster member"""
+        if (
+            hasattr(clustermember, "rfctobe_annotated")
+            and clustermember.rfctobe_annotated
+        ):
+            # Use the annotated field if available
+            return clustermember.rfctobe_annotated.get("disposition_slug")
+
+        doc = clustermember.doc
+
+        rfctobe = getattr(doc, "rfctobe_set", None)
+        if rfctobe is not None:
+            rfctobe = rfctobe.exclude(disposition__slug="withdrawn").first()
+
+        if not rfctobe:
+            return None
+
+        return rfctobe.disposition.slug
+
+    def get_references(self, clustermember: ClusterMember) -> list[dict] | None:
```

Comment:
Does this annotation result in a valid schema / API client types? I'm not sure what `list[dict]` will turn into when spectacular looks at it.